portfolio/src/main/webapp/script.js

```diff
@@ -88,3 +96,65 @@ function createListElement(text) {
   return liElement;
 }
 
+function loadContacts(){
+  fetch('/list-contacts')
+  .then(response => response.json())
+  .then((contacts) => {
+    const taskListElement = document.getElementById(contactPath);
+    console.log(contacts);
+    contacts.forEach((contact) => {
+      taskListElement.appendChild(createContactElement(contact));
+     
+    });
+  });
+}
+function createContactElement(contact) {
+  contactIds.push(contact.id);
+
+  const contactElement = document.createElement('li');
+  contactElement.className = contactLiteral;
+
+  const nameElement = document.createElement('span');
+  nameElement.innerText = nameLiteral + ": " + contact.name;
```

Comment:
See my comments above about this naming convention.

Reply:
I tried my best to fix this